apps/rewards/app/components/App/App.js

```diff
@@ -15,10 +15,20 @@ class App extends React.Component {
   static propTypes = {
     app: PropTypes.object.isRequired,
     rewards: PropTypes.arrayOf(PropTypes.object),
+    balances: PropTypes.arrayOf(PropTypes.object),
   }
 
   onNewReward = reward => {
-    console.log('Create New Reward from', reward)
+
+    this.props.app.newReward(
+      reward.rewardType ? false : true, //bool _isMerit,
```

Comment:
Can we adjust the rewardType so these are kept in sync, this seems like it would get confusing quick to have them inverted.